backend/src/v4/models/helper/model.js

```diff
@@ -560,11 +560,7 @@ function listSubModels(account, model, branch = "master") {
 			findModelSettingById(ref.owner, ref.project, { name: 1 }).then(subModel => {
 				// TODO: Why would this return null?
 				if (subModel) {
-					subModels.push({
-						database: ref.owner,
-						model: ref.project,
-						name: subModel.name
-					});
+					subModels.push(ref.project);
```

Comment:
is this right? `getModelSettings` also uses this function and as it stand we wouldn't be giving the submodel info that we used to give by changing this?